mne/time_frequency/tfr.py

```diff
@@ -99,6 +99,59 @@ def morlet(sfreq, freqs, n_cycles=7.0, sigma=None, zero_mean=False):
         Ws.append(W)
     return Ws
 
+def wavelet(sfreq, freqs, n_cycles=7.0, window=np.hanning):
```

Comment:
functions are mutable and mutable defaults are bad, along with the `get_window` change this can be changed to the immutable `'hann'`